deploy/operator.yaml

```diff
@@ -19,7 +19,6 @@ spec:
           image: REPLACE_IMAGE
           command:
           - apicast-operator
-          imagePullPolicy: Always
```

Comment:
When you create a project with the operator-sdk it automatically creates the deploy/operator.yaml with this configuration.
Is there a reason we want to remove this? Is it because you don't want the image to be always pulled?
We have the following tradeoffs:
 * If imagePullPolicy is not set then it will not always download the image (faster)
 * If imagePullPolicy is not set then if the  image tag is refreshed we won't get the new changes (will miss changes).

Reply:
three reasons:
* Apicast operator image will not be a floating tag, but fixed tag, so we know what is exact image running and we can manage upgrades.
* Testing "pushes" image to internal registry, so we need to be able to fetch from internal registry.
* As you know, default imagepullpolicy (IfNotPresent) will always download image if tag is latest or no tag is set. So we can mimic behavior if required. I think default policy is the most flexible and efficient one when using fixed tags.